src/components/Tabs.tsx

```diff
@@ -0,0 +1,36 @@
+import { Link, useParams } from 'react-router-dom';
+
+const Tabs = () => {
+  const { tabId } = useParams();
+  const tabs = [
+    { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
+    { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
+    { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+  ];
+
+  const isActive = (id: string) => (tabId === id ? 'is-active' : undefined);
```

Comment:
it's better set empty string if you use this value for class names
```suggestion
  const isActive = (id: string) => (tabId === id ? 'is-active' : '');
```